netlify.toml

```diff
@@ -54,4 +55,5 @@
   command="NEXT_PUBLIC_URL=$DEPLOY_URL npm run build"
   [context.deploy-preview.environment]
     NEXT_PUBLIC_URL="$DEPLOY_URL"
-    NEXT_PUBLIC_GTM_ID="GTM-TB4NLS7"
\ No newline at end of file
+    NEXT_PUBLIC_GTM_ID="GTM-TB4NLS7"
+    NEXT_PLUGIN_FORCE_RUN="false"
```

Comment:
When this gets approved, I will add one more commit to add this to our production environment.